app/controllers/transport/authorisationsAndLimit/authorisations/AddAnotherAuthorisationController.scala

```diff
@@ -34,6 +35,8 @@ import javax.inject.Inject
 class AddAnotherAuthorisationController @Inject() (
   override val messagesApi: MessagesApi,
   implicit val sessionRepository: SessionRepository,
+  navigatorProvider: TransportNavigatorProvider,
+  authNavigatorProvider: AuthorisationNavigatorProvider,
```

Comment:
I'm not massively keen on having two navigators in one controller but can see why it's been done. What we could do is create an `AuthorisationsNavigatorProvider` that uses the `AuthorisationsAndLimitDomain`